server/routers/outfit.py

```
@@ -300,7 +305,63 @@ async def suggest_outfit(
+    adventurousness = resolve_adventurousness(
+        req.adventurousness, style_ctx["adventurousness_default"]
+    )
+
+    upcoming_events: list = []
+    if req.scenario:
+        try:
+            calendar_source = get_calendar_source(client, scenario=req.scenario)
+            upcoming_events = await run_in_threadpool(
+                calendar_source.get_today_events, user_id
+            )
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)